packages/plugin-compiler/src/main/kotlin/io/realm/kotlin/compiler/IrUtils.kt

```diff
@@ -531,10 +531,15 @@ fun getBacklinksTargetPropertyType(declaration: IrProperty): IrType? {
     }
 }
 
-fun getLinkingObjectPropertyName(backingField: IrField): Name {
+fun getLinkingObjectPropertyName(property: IrProperty): String {
```

Comment:
I decided to change this to take `IrProperty` and return `String` to make this fit more general scenarios but I could revert that and try to do this a different way if this is not best behavior

Reply:
will need to change this back since it targets backingField again now